.github/renovate.json

```diff
@@ -45,5 +45,6 @@
       "matchManagers": ["github-actions"],
       "matchUpdateTypes": ["minor", "patch"]
     }
-  ]
+  ],
```

Comment:
ℹ️ If lockfiles show up for the .NET SDK here (I hope they don't) then we may need to force a Renovate constraint.